### ISSUE TITLE
🚀 Feature: Testimonials Page/Section

### ISSUE THREAD

**Issue by palveVaishnav** [code100x/job-board#282]
### 🔖 Feature description

# I want to work on the testimonials page/section

### 🎤 Why is this feature needed ?

- It is not there, so I though of adding it

### ✌️ How do you aim to achieve this?

Just want add it. 

### 🔄️ Additional Information

# I will try to build somethin like this 
![image](https://github.com/user-attachments/assets/78a88cc4-ff1d-4ed6-afe3-e33ea7cc6b7f)

- Add table to DB
- Add Zod Validations 
- Add some seed values too
- basically will try to implement everything related to it

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/code100x/job-board/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by palveVaishnav** [comment]
@VineeTagarwaL-code Can you assign this task to me !?
